WORKSPACE

```diff
@@ -236,7 +236,7 @@ filegroup(
     visibility = ["//visibility:public"],
 )
     """,
-    sha256 = "78400c82f1978147feeab54e3deccb29c024efa13d6753adc8687ae008bae246",
+    sha256 = "bb06d30ca533dc97d45f2367916ba9ff1b5af52f08a9d8c33bb7b1a61254094e",
     url = "https://github.com/ethereum/consensus-spec-tests/releases/download/%s/mainnet.tar.gz" % consensus_spec_version,
 )
 
```

Comment:
can you update sha256 at L254 for `consensus_spec`?


![Screenshot 2023-01-13 at 10 43 25 AM](https://user-images.githubusercontent.com/21316537/212395389-22012c36-8e45-43a3-8afd-864b673f3fbf.png)